app/services/mixtape_generator.rb

```diff
@@ -0,0 +1,20 @@
+class MixtapeGenerator
+  def initialize(genre:)
+    @genre = genre
+  end
+
+  def self.perform(genre:)
+    new(genre: genre).perform
+  end
+
+  def perform
+    Mixtape.new(genre: @genre, songs: pick_songs)
```

Comment:
Often have a `attr_reader` here for `genre` so don't have to use `@` for the ivar.